internal/lua-sandbox/api/api.go

```diff
@@ -0,0 +1,575 @@
+// Copyright The gittuf Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// This file contains modified code from the lua-sandbox project, available at
+// https://github.com/kikito/lua-sandbox/blob/master/sandbox.lua, and licensed
+// under the MIT License
+
+package sandbox
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	sandbox "github.com/gittuf/gittuf/internal/lua-sandbox/util"
+	lua "github.com/yuin/gopher-lua"
+)
+
+// Lua sandbox helper functions and wrapper functions of provided Go APIs
+var luaHelpers = `
```

Comment:
maybe make this a map of helper name -> helper implementation, then range through them and register them? seems cleaner to extend over time.